src/ccstruct/polyblk.cpp

```diff
@@ -34,7 +34,7 @@ namespace tesseract {
 
 #define INTERSECTING INT16_MAX
 
-int lessthan(const void *first, const void *second);
+int lessthan(const ICOORDELT *first, const ICOORDELT *second);
```

Comment:
I'll reconsider those functions. Maybe change to lambdas one-use funcs.

Reply:
It would be fine for if this is done later. If you want, the PR can be merged as soon as the missing include statements which currently break my local build were added. Should we squash the commits, or do you prefer to keep them all?

Reply:
The PR has 2-3 unrelated tiny fixes around msvc warnings, sw build.

Not sure what is better here.
I'd prefer to keep separate commits because of separate changes to each of three lists.